atox/src/main/res/layout/fragment_edit_user_profile.xml

```diff
@@ -0,0 +1,222 @@
+                android:layout_marginBottom="24dp">
+
+                <ImageView
+                    android:id="@+id/ic_status"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_alignParentStart="true"
+                    android:layout_centerVertical="true"
+                    android:contentDescription="@string/status_icon"
+                    android:src="@drawable/ic_status"
+                    app:tint="?attr/colorOnSurface" />
+
+                <com.google.android.material.textfield.TextInputLayout
+                    style="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox.ExposedDropdownMenu"
+                    android:id="@+id/edit_status"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="25dp"
+                    android:layout_toEndOf="@id/ic_status"
+                    android:hint="@string/status"
+                    android:textColorHint="@color/hint_text_color_day"
+                    app:startIconDrawable="@drawable/ic_available"
+                    app:boxStrokeColor="@color/box_stroke_color_day"
+                    app:endIconTint="@color/trailing_icon_color_day"
+                    app:hintTextColor="@color/hint_text_color_day">
+
+                    <AutoCompleteTextView
+                        android:id="@+id/edit_status_text"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:inputType="none"
+                        android:textAppearance="@style/TextAppearance.MaterialComponents.Subtitle1" />
+
+                </com.google.android.material.textfield.TextInputLayout>
+
+            </RelativeLayout>
+
+            <LinearLayout
+                android:id="@+id/edit_status_message"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:background="?android:attr/selectableItemBackground"
+                android:clickable="true"
+                android:focusable="true">
+                <RelativeLayout
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="32dp"
+                    android:layout_marginEnd="32dp"
+                    android:layout_marginVertical="24dp">
+
+                    <ImageView
+                        android:id="@+id/ic_status_message"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_alignParentStart="true"
+                        android:layout_centerVertical="true"
+                        android:contentDescription="@string/status_message_icon"
```

Comment:
Since these are purely cosmetic and not really needed for accessibility, I think you should drop these strings and just set the description to `@null`.

Reply:
I indeed prefer that option. Didn't know "@null" exists.